frontend/src/components/profile/ProfileStyles.js

```diff
@@ -0,0 +1,33 @@
+import BaseCard, { CardMedia as BaseCardMedia } from 'material-ui/Card'
+import styled from 'styled-components'
+import media from 'app/styleguide/media'
```

Comment:
@jardix22 so you're using the alias here, then I think it's just a matter to add in `webpack.config.production.js` as you done for development. The bundle is made on deploy, so it needs to make part of production config